release/rllib_tests/multi_node_checkpointing_compute_config.yaml

```diff
@@ -9,9 +9,9 @@ head_node_type:
 
 worker_node_types:
     - name: worker_node
-      instance_type: g4dn.xlarge
-      min_workers: 2
-      max_workers: 2
+      instance_type: g3.8xlarge
+      min_workers: 1
+      max_workers: 1
```

Comment:
@avnishn since you change the number of worker, you also need to change the number here https://github.com/ray-project/ray/blob/master/release/release_tests.yaml#L3679 to wait for the correct number of node

This broke this test, and some other rllib test in this latest run https://buildkite.com/ray-project/release-tests-branch/builds/1846

Please cherry pick the fix as well since the branch is cut

Reply:
I'm putting https://github.com/ray-project/ray/pull/37022 to fix